helper/inheritance-resolver.js

```diff
@@ -11,8 +11,17 @@ module.exports = function(plugins, config, name) { // eslint-disable-line func-n
   }
 
   function generateSymlinks(src, dest, replacePattern, ignore = []) {
+    var themeIgnoreFiles = [];
+    // Iterate through the config.themes to grab any ignored paths that
+    // themers want to be excluded from the gulp.src() and sass compilation
+    Object.keys(config.themes).forEach(name => {
```

Comment:
But I don't get the point of taking ignores from all themes and using them for a single theme.
Even in terms of inherited themes, if a parent ignores these files, there is no need to ignore them on the child level, b/c they not exist.

IMO this part should be removed and `config.themes[name].ignore` should be used in line 24 instead `themeIgnoreFiles`.